src/Servers/Kestrel/tools/CodeGenerator/TransportConnectionFeatureCollection.cs

```diff
@@ -20,7 +20,12 @@ public static string GenerateFile()
                 "IPersistentStateFeature",
                 "IMemoryPoolFeature",
                 "IConnectionLifetimeFeature",
-                "IConnectionSocketFeature"
+                "IConnectionSocketFeature",
+                "IProtocolErrorCodeFeature",
+                "IStreamDirectionFeature",
+                "IStreamIdFeature",
+                "IStreamAbortFeature",
+                "ITlsConnectionFeature"
```

Comment:
I wonder when it starts to make sense to generate different base classes for the Socket and Quic transports. This is still nowhere near HttpProtocol levels of features, so I suppose this is fine for now.